moveit_ros/moveit_servo/src/servo_server.cpp

```diff
@@ -96,7 +96,9 @@ bool ServoServer::init()
     planning_scene_monitor_->setPlanningScenePublishingFrequency(25);
     planning_scene_monitor_->startPublishingPlanningScene(planning_scene_monitor::PlanningSceneMonitor::UPDATE_SCENE,
                                                           "/moveit_servo/publish_planning_scene");
-    planning_scene_monitor_->startSceneMonitor();
+    planning_scene_monitor_->startSceneMonitor(
+        planning_scene_monitor::PlanningSceneMonitor::MONITORED_PLANNING_SCENE_TOPIC);
+    planning_scene_monitor_->requestPlanningSceneState();
```

Comment:
I don't understand what this line does. Are you using it to wait for an update?

If so, please add a comment to that effect:

`// Request a planning scene update and wait for the service response`